tools/ssm/ssm_create.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+set -eu
+
+PKG_NAME=$1
+REL_VERSION=$2
+S3_BUCKET=$3
+
+output=$(aws ssm list-documents --filter Key=Owner,Values=Self Key=DocumentType,Values=Package Key=Name,Values=${PKG_NAME})
+echo ${output}
+status=$(echo ${output} | python3 -c "import sys, json; print('empty') if len(json.load(sys.stdin)['DocumentIdentifiers']) == 0 else print('exist')")
```

Comment:
Q: do we need to check or pre-install if `python3 -c ` is not available on the target host?

Reply:
The python3 is listed in installed software of ubuntu-latest test runner as below.
https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md#installed-software